src/Character.js

```diff
@@ -60,6 +61,26 @@ class Character extends EventEmitter
    */
   getMaxAttribute(attr) {
     const attribute = this.attributes.get(attr);
+
+    // health and attackpower are calculated based off other stats
+    // NOTE: This is an _example_ implementation based off WoW formulas
+    switch (attribute.name) {
```

Comment:
This is a cool way to set up "calculated attributes" (those based off of other stats).